src/libraries/System.Collections/src/System/Collections/Generic/PriorityQueue.cs

```diff
@@ -96,7 +103,7 @@ public PriorityQueue(int initialCapacity, IComparer<TPriority>? comparer)
                 ? Array.Empty<(TElement, TPriority)>()
```

Comment:
I'd be fine dropping this check at this point and just always allocating, even if it's 0.  I don't think we do a similar check in List or Queue or Stack or others, since it's not particularly likely to have 0 passed in.

Reply:
Addressed.